src/subscriber_list.js

```diff
@@ -288,6 +328,180 @@ class SubscriberListsApi {
     }
   }
 
+  async delete(list_id) {
+    const cleaned_list_id = this._validate_list_id(list_id);
+    const content_text = JSON.stringify({});
+
+    const url = `${this.__subscriber_list_detail_url(
+        cleaned_list_id
+    )}delete/`;
+
+    const headers = { ...this.__headers, ...this.__dynamic_headers() };
+
+    const signature = get_request_signature(
+        url,
+        "PATCH",
+        content_text,
+        headers,
+        this.config.workspace_secret
+    );
+    headers["Authorization"] = `${this.config.workspace_key}:${signature}`;
+
+    try {
+      const response = await axios.patch(url,  content_text, { headers });
+      return response.data;
+    } catch (err) {
+      throw new SuprsendApiError(err);
+    }
+  }
+
+  async add_to_version(list_id, version_id, distinct_ids) {
+    const cleaned_list_id = this._validate_list_id(list_id);
+    const cleaned_version_id = this._validate_version_id(version_id)
+
+    if (!Array.isArray(distinct_ids)) {
+      throw new SuprsendError("distinct_ids must be list of strings");
```

Comment:
distinct ids can be ofany tipe not only string

Reply:
No, backend accepts only list of strings, see https://github.com/suprsend/suprsend-node-sdk/blob/643dd6f71d54292eb163288967d0df9ce7a46f91/src/subscriber_list.js#L267-L278 for ref